communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+# UNAM Engineering 
+[UNAM Engineering](https://github.com/unamfi) is a small Github organization started on 2016 with the objective to provide students a space
+where they were able to share the projects they have been developing on their classes and hackathons, tools and tips for other students. We
+promote students to go to meetups and various event, help them prepare for job/internship opportunities and provide office hours.
+
+### Definition
+
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in developing projects in their free time.
+  - Students already working on research and/or developing apps.
+  - Students interested to attend hackathons and meetups.
+  - Students interested in landing internship opportunities.
+  - Students interested in meeting industry companies and people.
+
+  
+### Size
+- How many people are currently in your community?
+
+Currently there's 45 people on the GitHub organization and around 10 people have submitted repos.
+
+- How many would you _like_ to be in your community?
+
+Getting to 150 people in the organization and having a 20% - 25% members contributing.
```

Comment:
This is a very high percentage of contributors, from a given community usually 10% is what can be expected. It's great to have a strong goal, but this may be hard, so don't get too discouraged!